components/VestingInput/index.tsx

```diff
@@ -408,7 +409,12 @@ function VestingInput({ refetch }: Props) {
                 <Trans
                   i18nKey="Not enough EXA for reserve. <1>Get EXA</1>."
                   components={{
-                    1: <Link href="/get-exa" style={{ fontWeight: 700, textDecoration: 'underline' }} />,
+                    1: (
+                      <a
+                        onClick={openGetEXA}
+                        style={{ fontWeight: 700, textDecoration: 'underline', cursor: 'pointer' }}
+                      />
```

Comment:
Is it possible to make this an unstyled button? An anchor with no href acts as a plain div